org-super-agenda.el

```diff
@@ -250,7 +252,13 @@ Prepended with `org-super-agenda-header-separator'."
                       ;; testing, it only takes effect in Agenda buffers when `local-map' is set, so
                       ;; we'll use both.
                       'local-map org-super-agenda-header-map)
-       (concat org-super-agenda-header-separator s))))
+       (concat
+        (cl-typecase org-super-agenda-header-separator
+          (character (concat (make-string (window-width)
+                                          org-super-agenda-header-separator)
```

Comment:
Nitpick: Having `org-super-agenda-header-separator` on a separate line may preserve 78-80 character width, but it's ugly to me, and I don't strictly follow that guideline, so please remove the newline.  :)

Reply:
Changed.